includes/views/upgrade-table.php

```diff
@@ -95,3 +95,39 @@
 		<td align="left" class="last">&nbsp;</td>
```

Comment:
Currently the bottom row of the features table can display the wrong amount of cells. This is because we present information here that is using the license data from the installed plugins. To prevent a call to our license server each time the Upgrade tab is opened we cache this data. So in my case there still was a cell displayed for the Premium Templates extension. We need to make sure this is skipped.

Reply:
@Terminator-Barbapapa I have added `true` parameter for ignore_cache parameter, I think it helps:
`$extension_license_infos = $this->get_extension_license_infos( true );`

Reply:
We still want to cache the license data. So that solution won't work. This will need to be handled in `upgrade-table.php` file, in the table row where the upgrade links are generated.

Reply:
@Terminator-Barbapapa I was able to reproduce your issue and I managed to solve it.